code/style.css

```diff
@@ -1,5 +1,237 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
-
 body {
-  background-color: lightblue;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+  
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  text-align: center;
+  color: white;
+}
+
+.centered {
+  position: absolute;
+  background-color: white;
+  color:#008542;
+  margin: 10px;
+  padding: 10px;
+  bottom: 10%;
+  width:65%;
+}
+
+@media min-width: 481px ) {
+  .centered {
+    margin: 20px;
+    padding: 20px;
+  }
 }
+
+
+h1 {
+font-size: 20px;
+font-style: normal;
+font-weight: 500;
+line-height: 1.2;
+text-align: center;
+color: #008542;
+  }
+
+  @media ( max-width: 768px ) and ( min-width: 481px ) {
```

Comment:
Great job on the media queries! 

You could add them like you've done here, after each item, but I would say it's more common to place all media queries in the bottom of the css-file. By doing so you only have to write the media-rule (`@media ( max-width: 768px ) and ( min-width: 481px ) {`) once, per size. 🙂